### PR TITLE
Update pod-security-policy.yaml remove formatting issues

### PR DESCRIPTION
**Pull request by 616b2f**
When we try to apply this file with kubectl apply it shows some formatting errors, this fixes them.

**Reason for Change**:

**Requirements**

- [x] squashed commits
- [ ] included documentation
- [ ] added unit tests and e2e tests (if applicable).

**Issue Fixed**:

**Does this change contain code from or inspired by another project?**

- [ ] Yes
- [ ] No

**If "Yes," did you notify that project's maintainers and provide attribution?**

**Special Notes for Reviewers**:

### CODE REVIEW

**Review by aramase — APPROVED** [review]

**Comment by nilekhc** [comment]
@616b2f Thanks for opening PR. Could you add e2e test for this? Currently we run e2e test by installing provider via helm charts.

We can modify [e2e_test_suite](https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/master/test/e2e/e2e_suite_test.go) to install provider from [deployment](https://github.com/Azure/secrets-store-csi-driver-provider-azure/tree/master/deployment) dir and run entire suite.

File: deployment/pod-security-policy.yaml [hunk 1]
```diff
@@ -66,9 +66,9 @@ spec:
   - hostPath
   - secret
   hostNetwork: true
-  hostPort:
+  hostPorts:
   - min: 0
-  - max: 65535
+    max: 65535
```
Comment (nilekhc) [inline deployment/pod-security-policy.yaml:71]: Could you add e2e test for this? Currently we run e2e test by installing provider via helm charts.

We can modify [e2e_test_suite](https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/master/test/e2e/e2e_suite_test.go) to install provider from [deployment](https://github.com/Azure/secrets-store-csi-driver-provider-azure/tree/master/deployment) dir and run entire suite.

Comment (aramase) [reply deployment/pod-security-policy.yaml:71]: The e2e tests are being tracked here: https://github.com/Azure/secrets-store-csi-driver-provider-azure/issues/196

**Review by nilekhc — APPROVED** [review]
Already tracking issue to add e2e. 

lgtm

**Pull request merged**